porcupine/plugins/text_google_search.py

```diff
@@ -0,0 +1,21 @@
+"""Search selected text on Google."""
+
+import tkinter as tk
+from porcupine import menubar, tabs
+import webbrowser
+
+def google_search(tab: tabs.FileTab) -> None:
+    try:
+        selected_text_tuple = tab.textwidget.tag_ranges(tk.SEL)
+        # Check if text is selected to search on google
+        if len(selected_text_tuple)>0:
+            selected_text = tab.textwidget.get(*selected_text_tuple)
+            url = "https://www.google.com/search?q={}".format(selected_text)
```

Comment:
Shouldn't this quote the text somehow? For example, if you google search for `std::vector<int>& values`, the URL in browser starts with `https://www.google.com/search?q=std%3A%3Avector%3Cint%3E%26+values`, not `https://www.google.com/search?q=std::vector<int>& values`.

I think you can use `urllib.parse.quote_plus()` for this:

```
>>> import urllib.parse
>>> urllib.parse.quote_plus('hello world&foo=bar')
'hello+world%26foo%3Dbar'
```

Reply:
Resolved in commit https://github.com/Akuli/porcupine/pull/1024/commits/0869a0e826a7438a3864e9cb4fff3b3346577310